spec/unleash/strategy/email_domain_spec.rb

```diff
@@ -0,0 +1,95 @@
+require 'spec_helper'
+
+RSpec.describe Unleash::Strategy::EmailDomain do
+  describe 'is_enabled?' do
+    subject(:is_enabled?) { described_class.new.is_enabled?(params, context) }
+    let(:context) do
+      Unleash::Context.new(
+        properties: { user_email: 'rafael.albuquerque@inloco.com.br' }
+      )
+    end
+    let(:params) { { 'emailDomains' => 'inloco.com.br,inloco.com' } }
+
+    context 'when params is invalid' do
+      let(:expected_result) { false }
```

Comment:
I think that this let is not being used, right?

The same to the rest

Reply:
Sorry, I just saw this after merge. :sleepy: 

https://github.com/inloco/unleash-incognia-ruby/pull/2